index.js

```diff
@@ -683,4 +683,64 @@ Peer.prototype._debug = function () {
   debug.apply(null, args)
 }
 
+// Transform constraints objects into the new format (unless Chromium)
+// This can be removed when Chromium supports the new format
+function transformConstraints (constraints) {
+  if (typeof (window) === 'undefined' || Object.keys(constraints).length === 0) {
+    return constraints
+  }
+  var isChromium = window.chrome
```

Comment:
Can you move this to line 83, near the other user agent sniffing code? https://github.com/feross/simple-peer/blob/master/index.js#L83 It's better for all the user agent sniffing hacks to be in one place, IMO.